Assets/Scripts/Interactable.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 5b73ba88e606fd44185a5927b45fba57
+guid: acfb641981c1740e4ac7242489e31d62
```

Comment:
If i'm interpreting this diff correctly, the wwise meta file got deleted. Assuming that was intended/

Reply:
Wwise meta is ignored

I didn't deliberately delete any - the working directory i was handed had a large diff, and believe this malarky was caused by the files being so similar that Git interpreted these changes as a rename+modification.

With a quick readup i found
> The files are similar enough so that git status thinks that it was a rename

> Git infers renames when it detects deletions and additions of similar content (by default 50% similar) whenever diffs are computed between any pair of commits in the history

> Git's logical underlying storage model only stores the repo contents before and after a change, not the change itself. So it has no way of distinguishing between, say, a move+modification and a delete+add.

Reply:
Righto. So some wwise metafiles must have slipped in before they got ignored. We should probably do a sweep of those files sometime

Thanks for clarifying!